### ISSUE TITLE
Investigate WCAG Checker

### ISSUE THREAD

**Issue by helenvholmes** [NYPL/nypl-design-system#108]
Using aXe, either checking the code or as a Storybook integration. Which works best for us?

**Comment by M0nica** [comment]
@scooooooooby @hokei  I think we should add Axe-Linter to the project to ensure that pull requests are linted for a11y related issues. We are already using the Axe storybook integration within NYPL Design Systems so I think the Axe-Linter would compliment it well. 

We can set up the linter to check the changes within each pull request against a specific version of WCAG requirements. If any issues arise it will result in a `fail` status check. 

Do we want the listing to happen on all files? If not, what files/types of files should be excluded? I am thinking to exclude files that are not  `.ts` `.tsx` or `.stories` .

You can view the GitHub app here: https://github.com/marketplace/axe-linter

**Comment by M0nica** [comment]
The linter should be set to check the code against the WCAG 2.1 AA specification.

**Comment by helenvholmes** [comment]
@M0nica that sounds like a great improvement to me.

Your ideas on what to include/not include make sense to me as well.

**Comment by M0nica** [comment]
Closing this issue as an issue was added to implement this functionality https://github.com/NYPL/nypl-design-system/issues/177
